components/NewsroomYoutube.js

```diff
@@ -0,0 +1,91 @@
+import { useEffect, useState, useRef } from 'react';
+import TextTruncate from 'react-text-truncate'
+import ArrowLeft from './icons/ArrowLeft';
+import ArrowRight from './icons/ArrowRight';
+import TextLink from './typography/TextLink';
+import tempData from '../config/videos.json';
+import Heading from './typography/Heading'
+import Paragraph from './typography/Paragraph'
+
+export default function NewsroomYoutube({ className = ''}) {
+  //console.log(data);
+  const [videoData, setvideoData] = useState([]);
+  const videoContainer = useRef();
+  const videoref = useRef();
+  const shiftLeft = (e) => {
+    e.preventDefault();
+    const width = videoref.current.clientWidth;
+    videoContainer.current.scrollLeft -= width + 16;
+  };
+  const shiftRight = (e) => {
+    e.preventDefault();
+    const width = videoref.current.clientWidth;
+    videoContainer.current.scrollLeft += width + 16;
+  };
+
+  let buttonClass = 'bg-secondary-100 hover:bg-secondary-500 shadow-md rounded border-secondary-500 border text-secondary-500 hover:text-white focus:outline-none mx-2';
+  useEffect(() => {
+    async function youtube(){  
+      await fetch('/.netlify/functions/youtube_videos').then((response)=>{
+        //response.json();
+        console.log(response)
```

Comment:
@derberg The response body from the netlify functions is not getting rendered here. Netlify functions are not returning any body text within, although the body has been returned from the function. You can check by adding `YOUTUBE_TOKEN` in the `.env` file and run the command `netlify dev`. 

**Note** - If `netlify dev` doesn't work, try first installing netlify-cli globally using:
```
npm i netlify-cli -g
```
and then use the command `netlify dev`.